CHANGELOG.md

```diff
@@ -106,8 +115,10 @@
 * [ENHANCEMENT] The query-tee node port (`$._config.query_tee_node_port`) is now optional. #3272
 * [ENHANCEMENT] Add support for autoscaling distributors. #3378
 * [ENHANCEMENT] Make auto-scaling logic ensure integer KEDA thresholds. #3512
+* [ENHANCEMENT] Refactor read-write-deployment jsonnet. #3405
```

Comment:
Just add this PR number to the CHANGELOG entry "[FEATURE] Added support for experimental read-write deployment mode"

Reply:
Fixed: 9403c0e75d32c45e6ae446e2f11e20ce168ebb45